packages/app/src/sections/dashboard/DashboardLayout.tsx

```diff
@@ -61,24 +60,24 @@ const DashboardLayout: FC<{ children?: ReactNode }> = ({ children }) => {
 				disabled: false,
 				href: ROUTES.Dashboard.Markets,
 			},
+			{
+				name: Tab.Governance,
+				label: t('dashboard.tabs.governance'),
+				active: activeTab === Tab.Governance,
+				href: EXTERNAL_LINKS.Docs.Governance,
+				external: true,
+			},
 			{
 				name: Tab.Stake,
 				label: t('dashboard.tabs.staking'),
 				active: activeTab === Tab.Stake,
 				href: ROUTES.Dashboard.Stake,
 			},
 			{
-				name: Tab.Rewards,
-				label: t('dashboard.tabs.rewards'),
-				active: activeTab === Tab.Rewards,
-				href: ROUTES.Dashboard.Rewards,
-			},
-			{
-				name: Tab.Governance,
-				label: t('dashboard.tabs.governance'),
-				active: activeTab === Tab.Governance,
-				href: EXTERNAL_LINKS.Docs.Governance,
-				external: true,
+				name: Tab.Migrate,
+				label: 'Migrate to V2',
```

Comment:
i18n

Reply:
Thanks for the reminder. This is a temporary label on the sidebar for preview purposes. I will remove it before the final release.